doc/api/single-executable-applications.md

```diff
@@ -81,13 +81,14 @@ tool, [postject][]:
 
    * On systems other than macOS:
      ```console
-     $ npx postject hello NODE_SEA_BLOB sea-prep.blob \
+     npx postject hello NODE_SEA_BLOB sea-prep.blob \
          --sentinel-fuse NODE_SEA_FUSE_fce680ab2cc467b6e072b8b5df1996b2
      ```
 
    * On macOS:
      ```console
-     $ npx postject hello NODE_SEA_BLOB sea-prep.blob \
+     chmod u+rw hello ;# make file writable
```

Comment:
Is the `chmod` required on macOS? I'm able to build the SEA on macOS without explicitly running the `chmod` command.

Reply:
I needed it